normandy/conftest.py

```diff
@@ -0,0 +1,23 @@
+# Configuration file for running our tests
+import pytest
+
+TIMEOUT = 20
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        '--bin',
+        default='/Applications/FirefoxBeta.app/Contents/MacOS/firefox-bin',
+        help='path for Firefox binary',
```

Comment:
the user should know this is only for 1 OS.  perhaps something like:
help='path to Firefox binary (OSX only)'